### PR TITLE
Feature/comlet/integration systemtest

### PR DESCRIPTION
**Pull request by AnNeufang**
systemtest jobs are ready for integration
please review and take care, that the schedule for develop must be adapted to run the jobs in Nightly Build

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -434,3 +454,81 @@ develop-test-qemu:
...
+    - "develop-build-deploy-distro-targets"
+  before_script:
+    - docker login ${CI_REGISTRY} -u ${SYSTEMTEST_DOCKER_REG_USER} -p ${SYSTEMTEST_DOCKER_REG_TOKEN}
+  script:
+    - echo "checkout systemtest repository"
+    - BASE_URL=`echo $CI_REPOSITORY_URL | sed "s;\/*$CI_PROJECT_PATH.*;;"`
+    - REPO_URL="$BASE_URL/rd-t/irma-six/systemtest.git"
+    - git clone $REPO_URL $SYSTEMTEST_PROJECT_DIR
+    - cd $SYSTEMTEST_PROJECT_DIR
+    - git checkout $SYSTEMTEST_BASE
```
Comment (martin31821) [inline .gitlab-ci.yml:484]: These parts seem to be duplicated. I think I'd move them into a template and move them to before_script.

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -434,3 +454,81 @@ develop-test-qemu:
...
+    - "systemtest_smoke"
+  before_script:
+    - docker login ${CI_REGISTRY} -u ${SYSTEMTEST_DOCKER_REG_USER} -p ${SYSTEMTEST_DOCKER_REG_TOKEN}
+  script:
+    - echo "checkout systemtest repository"
+    - BASE_URL=`echo $CI_REPOSITORY_URL | sed "s;\/*$CI_PROJECT_PATH.*;;"`
+    - REPO_URL="$BASE_URL/rd-t/irma-six/systemtest.git"
+    - git clone $REPO_URL $SYSTEMTEST_PROJECT_DIR
+    - cd $SYSTEMTEST_PROJECT_DIR
+    - git checkout $SYSTEMTEST_BASE
```
Comment (martin31821) [inline .gitlab-ci.yml:523]: same as above

**Review by Jasper-Ben — APPROVED** [review]
Other than Martins comments this looks good to me :+1:

**Comment by Jasper-Ben** [comment]
Before this is merged, could you please squash into a single commit with meaningful commit msg? Thanks!

**Comment by Jasper-Ben** [comment]
I've addressed Martin's comment in https://github.com/iris-GmbH/iris-kas/pull/143/commits/d98e28868d584ccc5fafcbb8f548198baf72ade7. Also, As the reliability test depends on the smoke test, skipping the latter will now also skip the reliability test. Please review and squash if happy with these changes @AnNeufang.

**Comment by AnNeufang** [comment]
@Jasper-Ben Sqash inclusice cleanup update performed

**Review by martin31821 — APPROVED** [review]
LGTM

**Pull request merged**